test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoDeleteSuite.java

```diff
@@ -87,13 +93,56 @@ private HapiApiSpec cannotDeleteAccountsWithNonzeroTokenBalances() {
 		return defaultHapiSpec("CannotDeleteAccountsWithNonzeroTokenBalances")
 				.given(
 						cryptoCreate("toBeDeleted"),
-						cryptoCreate("transferAccount")
+						cryptoCreate("transferAccount"),
+						cryptoCreate(TOKEN_TREASURY)
 				).when(
-						tokenCreate("misc").treasury("toBeDeleted")
+						tokenCreate("misc").initialSupply(INITIAL_FLOAT).treasury(TOKEN_TREASURY),
+						tokenAssociate("toBeDeleted", "misc"),
+						tokenTransact(HapiTokenTransact.TokenMovement.moving(INITIAL_FLOAT, "misc").between(TOKEN_TREASURY, "toBeDeleted"))
 				).then(
 						cryptoDelete("toBeDeleted")
 								.transfer("transferAccount")
 								.hasKnownStatus(TRANSACTION_REQUIRES_ZERO_TOKEN_BALANCES)
 				);
 	}
+
+	private HapiApiSpec deleteFailureScenarios() {
+		long B = HapiSpecSetup.getDefaultInstance().defaultBalance();
+
+		return defaultHapiSpec("DeleteFailureScenarios")
```

Comment:
There are other "failure scenarios" in the "CryptoDeleteSuite" now, so I think it would be better if they are separate tests. This test checks for several error cases in one test and it is a better practice if every test has a purpose